lib/paper_trail/events/base.rb

```diff
@@ -44,6 +57,13 @@ def changed_notably?
 
       private
 
+      # @api private
+      def assert_metadatum_key_is_permitted?(key)
```

Comment:
technically not a boolean method (if you want to follow the convention)

```suggestion
      def assert_metadatum_key_is_permitted(key)
```

Reply:
Good catch :)